### DISCUSSION TITLE
questions about dockerfile

### DISCUSSION THREAD

**Discussion by chiptus** [PhilDL/remix-gospel-stack#77] [Announcements]
1. why do we install openssl? removing it seems ok, but I don't want to remove without knowing.
2. why do we have two `pnpm run genrate`? 

Thanks!

**Comment by PhilDL** [comment]
Hi @chiptus 

- openssl is for Prisma, to generate the target binaries. For the longest time, I habitually added it on `***-slim` Docker base node image because it was not available. Maybe now it's included? I may have missed things.
- Honestly, I don't remember why there are 2 layers with prisma generate sorry, this Dockerfile is the result of a lot of trial and errors and changes. I would be glad if you could PR with optimization, Docker is not my expertise 😅
